src/components/external/DisputesManagement/components/DisputesData/DisputeDataProperties.tsx

```diff
@@ -0,0 +1,130 @@
+import cx from 'classnames';
+import { useMemo } from 'preact/hooks';
+import { TranslationKey } from '../../../../../translations';
+import { IDisputeDetail } from '../../../../../types/api/models/disputes';
+import CopyText from '../../../../internal/CopyText/CopyText';
+import Icon from '../../../../internal/DataGrid/components/Icon';
+import { isCustomDataObject } from '../../../../internal/DataGrid/components/TableCells';
+import Link from '../../../../internal/Link/Link';
+import StructuredList from '../../../../internal/StructuredList';
+import { StructuredListProps } from '../../../../internal/StructuredList/types';
+import { TypographyVariant } from '../../../../internal/Typography/types';
+import Typography from '../../../../internal/Typography/Typography';
+import type { TransactionDataProps } from '../../../TransactionDetails';
+import { TX_DATA_LABEL, TX_DATA_LIST, TX_DETAILS_RESERVED_FIELDS_SET } from '../../../TransactionDetails/components/constants';
+
+type DisputeDataPropertiesProps = {
+    dispute: IDisputeDetail;
+    dataCustomization?: TransactionDataProps['dataCustomization'];
+    extraFields?: Record<any, any>;
+};
+
+const DisputeDataProperties = ({ dispute, dataCustomization, extraFields }: DisputeDataPropertiesProps) => {
+    return useMemo(() => {
+        const { paymentPspReference, latestDefense, reasonCode, paymentMerchantReference, reasonGroup } = dispute;
+        const SKIP_ITEM: StructuredListProps['items'][number] = null!;
+
+        const disputeReasonKey = 'dispute.disputeReason';
+        const reasonCodeKey = 'dispute.reasonCode';
+        // const accountKey = 'dispute.account';
+        const disputeReferenceKey = 'dispute.disputeReference';
+        const paymentReferenceKey = 'dispute.paymentReference';
+        const merchantReferenceKey = 'dispute.merchantReference';
+        const defenseReasonKey = 'dispute.merchantReference';
+        const defendedOnKey = 'dispute.defendedOn';
+        const evidenceKey = 'dispute.evidence';
+
+        const listItems: StructuredListProps['items'] = [
+            // balance account
+            // balanceAccount?.description ? { key: 'accountKey' as const, value: balanceAccount.description, id: 'description' } : SKIP_ITEM,
+
+            // disputeReason
+            disputeReasonKey ? { key: disputeReasonKey as TranslationKey, value: reasonGroup, id: 'reasonGroup' } : SKIP_ITEM,
```

Comment:
The `as TranslationKey` type assertion not needed here.
```suggestion
            disputeReasonKey ? { key: disputeReasonKey, value: reasonGroup, id: 'reasonGroup' } : SKIP_ITEM,
```